### ISSUE TITLE
Allow password to be entered by pressing the 'ENTER' key on keyboard.

### ISSUE THREAD

**Issue by osBins** [akshgpt7/MarvellOS#11]
Currently, password can only be submitted by clicking confirm button on log-in screen, and 'enter' key does not work.

**Comment by sneaky-potato** [comment]
Hi. I think I can solve this issue. Could you please assign this to me

**Comment by akshgpt7** [comment]
Go for it @4shw4n1.

**Comment by ConsultantFoodie** [comment]
Sorry, but I didn't read that the issue was assigned. I have already made a PR for this.

**Comment by akshgpt7** [comment]
@4shw4n1 Did you also already complete this?
If not, maybe you can look for another issue to work on?
And @ConsultantFoodie thanks for the PR (I'll test it and merge it), but please make sure to comment on the issue before starting to work on it, since it helps others know that you're working on it and avoids such clashes :)

**Comment by akshgpt7** [comment]
Fixed via #14.
